public/js/main.js

```diff
@@ -0,0 +1,63 @@
+(function () {
+  var url = 'https://nazareth-open-tourism-platform.herokuapp.com/' + 'events';
```

Comment:
url could be hardcoded as `https://nazareth-open-tourism-platform.herokuapp.com/events`